Assets/Scripts/Nathan/Animator/CameraController.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
```

Comment:
Is this file used at all? I couldn't find anything with it either. If not being used, delete.

Reply:
I thought I had already deleted this